### PR TITLE
Ease up on the "Compute Saved" number animation.

### PR DESCRIPTION
**Pull request by anthonyshew**
### Description

Vercel Remote Cache has seen some exciting growth! However, it makes this number jump around a lot.

Using minutes as the unit was too loud so I moved it to hours. I also did some slight redesigning to give it more emphasis on the time unit.

### Testing Instructions

Take a look at the number in the sidebar. You'll see it go up still - but it won't be quite as obnoxious.

Closes TURBO-2174

### CODE REVIEW

File: docs/components/RemoteCacheCounter.tsx [hunk 1]
```diff
@@ -8,37 +8,44 @@ const counterFormatter = Intl.NumberFormat(undefined, {
   maximumFractionDigits: 0,
 });
 
+const START_NUM = 2080300;
```
Comment (chris-olszewski) [inline docs/components/RemoteCacheCounter.tsx:11]: Explanation of this number would be appreciated, it looks very magic to me.

**Review by chris-olszewski — APPROVED** [review]

File: docs/components/RemoteCacheCounter.tsx (hunk 1 above)
Comment (anthonyshew) [reply docs/components/RemoteCacheCounter.tsx:11]: Oops, deleted the comment when I was messing around and forgot to bring it back. Thanks.

Comment (anthonyshew) [inline docs/components/RemoteCacheCounter.tsx:11]:
```suggestion
// Arbitrary number based on the value on January 25th, 2024.
// If the number takes too long to update on page load, you can bump this number up.
const START_NUM = 2080300; 
```

File: docs/components/RemoteCacheCounter.tsx [hunk 2]
```diff
@@ -8,37 +8,46 @@ const counterFormatter = Intl.NumberFormat(undefined, {
...
       <div className="p-4 dark:bg-[#111111] bg-white">
-        <animated.p className="inline-block text-xl text-transparent bg-gradient-to-r from-red-500 to-blue-500 bg-clip-text tabular-nums">
-          {spring.minutesSaved.to((t) => counterFormatter.format(t))}
-        </animated.p>
-        <div className="text-xs">Total Compute Minutes Saved</div>
+        <span className="inline-flex gap-1 text-xl text-transparent bg-gradient-to-r from-red-500 to-blue-500 bg-clip-text">
+          <animated.p className="inline-block tabular-nums">
+            {spring.hoursSaved.to((t) => counterFormatter.format(t))}
+          </animated.p>
+          <p className="inline-block">hours</p>
```
Comment (tknickman) [inline docs/components/RemoteCacheCounter.tsx:44]: The hours shouldn't be gradient - only the number.

**Review by tknickman — CHANGES_REQUESTED** [review]

**Review by tknickman — APPROVED** [review]
I changed when I shoulda approved. Fix/ship - up to you!

**Pull request merged**